spot_driver/src/spot_driver/spot_wrapper.py

```diff
@@ -674,6 +695,240 @@ def navigate_to(self, upload_path,
+                    root_frame_name=ODOM_FRAME_NAME, wrench_trajectory_in_task=trajectory,
+                    x_axis=arm_command_pb2.ArmCartesianCommand.Request.AXIS_MODE_FORCE,
+                    y_axis=arm_command_pb2.ArmCartesianCommand.Request.AXIS_MODE_FORCE,
+                    z_axis=arm_command_pb2.ArmCartesianCommand.Request.AXIS_MODE_FORCE,
+                    rx_axis=arm_command_pb2.ArmCartesianCommand.Request.AXIS_MODE_FORCE,
+                    ry_axis=arm_command_pb2.ArmCartesianCommand.Request.AXIS_MODE_FORCE,
+                    rz_axis=arm_command_pb2.ArmCartesianCommand.Request.AXIS_MODE_FORCE)
+                arm_command = arm_command_pb2.ArmCommand.Request(
+                    arm_cartesian_command=arm_cartesian_command)
+                synchronized_command = synchronized_command_pb2.SynchronizedCommand.Request(
+                    arm_command=arm_command)
+                robot_command = robot_command_pb2.RobotCommand(
+                    synchronized_command=synchronized_command)
+
+                # Send the request
+                self._robot_command_client.robot_command(robot_command)
+                self._logger.info('Force trajectory command sent')
+
+                time.sleep(10.0)
+
+        except Exception as e:
+            return False, "Exception occured during arm movement" + str(e)
+        
+    def gripper_open(self):
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
+            else:
+                # Open gripper
+                command = RobotCommandBuilder.claw_gripper_open_command()
+
+                # Command issue with RobotCommandClient
+                self._robot_command_client.robot_command(command)
+                self._logger.info("Command gripper open sent")
+                time.sleep(2.0)
+
+        except Exception as e:
+            return False, "Exception occured while gripper was moving"
+
+        return True, "Open gripper success"
+
+    def gripper_close(self):
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
+            else:
+                # Close gripper
+                command = RobotCommandBuilder.claw_gripper_close_command()
+
+                # Command issue with RobotCommandClient
+                self._robot_command_client.robot_command(command)
+                self._logger.info("Command gripper close sent")
+                time.sleep(2.0)
+                
+        except Exception as e:
+            return False, "Exception occured while gripper was moving"
+
+        return True, "Closed gripper successfully"
+    
+    
+    def gripper_angle_open(self, gripper_ang):
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
```

Comment:
```suggestion
                self._logger.info(msg)
                return False, msg
```

Otherwise this method will return `True, "Opened gripper successfully` on line 927 because no Exception will be thrown. 
The same applies to all other methods. We'll need an early exit there as well.